wallet/bench/Client/Pos/Wallet/Web/Api.hs

```diff
@@ -0,0 +1,52 @@
+{-# LANGUAGE TypeOperators #-}
+
+-- | Client functions for Wallet Web API. To be more precise,
+-- only for endpoints we have to benchmark.
+
+module Client.Pos.Wallet.Web.Api
+    ( getHistory
+    , getWallets
+    , newPayment
+    ) where
+
+import           Universum
+
+import           Servant.API                ((:<|>) (..), (:>))
+import           Servant.Client             (ClientM, client)
+
+import           Pos.Client.Txp.Util        (InputSelectionPolicy)
+import           Pos.Core.Types             (Coin)
+import           Pos.Wallet.Web.Api         (ApiPrefix, GetHistory, GetWallets,
+                                             NewPayment)
+import           Pos.Wallet.Web.ClientTypes (Addr, CAccountId (..),
+                                             CId (..), CPassPhrase, CTx, CWallet,
+                                             ScrollLimit, ScrollOffset, Wal)
+
+-- | "Benchmarking API" which includes
+-- endpoints we need for benchmarking.
+type WalletBenchApi = ApiPrefix :> (
```

Comment:
Do we really need to use `ApiPrefix` here? what does it buy us? (Genuine question).

Reply:
Yes, we need it. Because this `ApiPrefix` is using in the complete API, so I want to use it here as well. Yes, we're using just 3 endpoints, but it should be the **same** as if _all_ API contained only these 3 endpoints.